### ISSUE TITLE
The Python example on the Search Metrics documentation page is incorrect

### ISSUE THREAD

**Issue by gsalami00** [DataDog/documentation#5556]
**Describe the bug**
The Python example on the [Search Metrics documentation page](https://docs.datadoghq.com/api/?lang=python#search-metrics) is incorrect.

The example listed is:

```python
from datadog import initialize, api

options = {
    'api_key': '<YOUR_API_KEY>',
    'app_key': '<YOUR_APP_KEY>'
}

initialize(**options)

api.Hosts.search(q="metrics:system")
```

It should be:

```python
from datadog import initialize, api

options = {
    'api_key': '<YOUR_API_KEY>',
    'app_key': '<YOUR_APP_KEY>'
}

initialize(**options)

api.Infrastructure.search(q="metrics:test")
```

I'd like to submit a pull request. Let me know how to proceed. :)

**Comment by l0k0ms** [comment]
Hello, 

Many thanks for the feedback 🙇 
After looking into this, it seems that this endpoint is indeed deprecated: 
https://github.com/DataDog/datadogpy/blob/master/datadog/api/infrastructure.py#L23

But the `api.host.search()` only covers half the use case, since it returns only the hosts from a Datadog organization: https://github.com/DataDog/datadogpy/blob/master/datadog/api/infrastructure.py#L15

This endpoint is deprecated since it's covered by the host search and the metric search endpoints: 

* https://docs.datadoghq.com/api/?lang=python#get-list-of-active-metrics
* https://docs.datadoghq.com/api/?lang=python#search-hosts

I think the PR here would be to add a deprecation notice over this endpoint to advise users not to use it and use the host/metric search endpoints instead while linking to them.

To contribute to the documentation repository, you can find the setup instructions for a local documentation build [here](https://github.com/DataDog/documentation#setup) and the contribution workflow [here](https://github.com/DataDog/documentation#working-on-docs).

But for a small change like this one you could directly edit the file through the GitHub UI: https://github.com/DataDog/documentation/blob/master/content/en/api/metrics/metrics_search.md

For this change you can inspire yourself with the Screenboard endpoint deprecation: https://docs.datadoghq.com/api/?lang=python#screenboards 
[Source code for the Screenboard endpoint](https://github.com/DataDog/documentation/blob/master/content/en/api/screenboards/screenboards.md)

What do you think ? :) 
Best regards,
Pierre

**Comment by l0k0ms** [comment]
Hello, 

Thanks for the feedback, after looking into it, it seems that only the `host:` metric prefix is deprecated, i opened a PR to reflect more closely the behavior of this endpoint + fixed the examples: https://github.com/DataDog/documentation/pull/5573

**Comment by apigirl** [comment]
fixed by #5573
